httpie/output/utils.py

```diff
@@ -0,0 +1,54 @@
+
+def parse_header(line):
+    """Parse a Content-Type like header.
+    Return the main Content-Type and a dictionary of options.
```

Comment:
This looks like a specialized function with a very generic name. Something like `parse_structured_header()` or even `parse_content_type_header_overwrite()` would work better.